snowpack/src/commands/dev.ts

```diff
@@ -166,7 +166,7 @@ const sendFile = (
   }
 
   res.writeHead(200, headers);
-  res.write(body, getEncodingType(ext) as BufferEncoding);
+  res.write(body);
```

Comment:
This is a benefit of this approach: by not caring about encoding types, we’re less likely to get it wrong.